data/strings/types_strings.txt

```diff
@@ -21161,7 +21167,7 @@
     eu = Margoak
     fa = ﺪﻨﮐ ﯽﻣ ﮓﻧﺭ
     fi = Maalit
-    fr = Des peintures
+    fr = Magasin de peinture
```

Comment:
Should we include "Magasin" (e.g. "shop" synonyms) in these translations? Users expect to type and search names that they see in the Place Page.

Reply:
This is a types_strings - what user see in PP, so "Magasin" is ok here.
categories.txt has the same string now, can't say is it good or not, according to French grammar.

Anyway, this is an example why we should _carefully_ treat translations as search categories, not straight-forward.